### PR TITLE
Add more detailed instructions for consumer apps

### PR DESCRIPTION
**Pull request by frenchwr**
This adds a bit more detail to the README. I also updated the CI to only kick off when something about the snap itself changes (not just the README).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,23 +104,17 @@ Typical output:
...
 
 ```
 sudo usermod -a -G render $USER # log out and log back in
 ```
 
-If this is your first run, or if you re-loaded the `intel_vpu` driver, then you'll also need to perform the following:
+Next apply the appropriate permissions on the device node. This is required each time the `intel_vpu` driver is reloaded, e.g. when the snap is first installed or following a reboot.
 
 ```
 sudo chown root:render /dev/accel/accel0
```
Comment (hector-cao) [inline README.md:120]: The `usermod` and `chown` instructions are necessary only if the user wants to run `vpu-umd-test` in no-root mode right ?

Comment (frenchwr) [reply README.md:120]: @hector-cao Yes, that's correct.

Comment (hector-cao) [reply README.md:120]: by curiosity, what are the default ownership/permissions settings of `/dev/accel/accelxx` devices ?

**Review by hector-cao — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (frenchwr) [reply README.md:120]:
```
$ ls -lh /dev/accel/accel0
crw------- 1 root root 261, 0 Oct 16 15:27 /dev/accel/accel0
```

**Pull request merged**